api/src/main/java/org/openmrs/util/Security.java

```diff
@@ -33,14 +33,16 @@
  * OpenMRS's security class deals with the hashing of passwords.
  */
 public class Security {
-
-	private Security() {
-	}
+	
+	/**
+	 * Defined encoding to avoid using default platform charset 
+	 */
+	private static final String encoding = "UTF-8";
 	
 	/**
 	 * encryption settings
 	 */
-	private static final Logger log = LoggerFactory.getLogger(Security.class);
+	public static Logger log = LoggerFactory.getLogger(Security.class);
```

Comment:
Did you change the above from private to public, accidentally?

Reply:
I didn't touch this file, it's autogenerated.

Reply:
Then do not include it when committing.

Reply:
Okay, let me remove it